pkgs/tools/bluetooth/bluewalker/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitLab }:
+
+buildGoModule rec {
+  pname = "bluewalker";
+  version = "0.3.0";
+
+  src = fetchFromGitLab {
+    owner = "jtaimisto";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-spuJRiNiaBV4EsetUq8vUfR6ejUNZxLhVzS3AZZyrrQ=";
+  };
+
+  vendorSha256 = "189qs6vmx63vwsjmc4qgf1y8xjsi7x6l1f5c3kd8j8jnagl26z4h";
+
+  ldflags = [ # omit some things for a smaller binary
+    "-w" # omit the DWARF symbol table
+    "-s" # omit symbol table and debug info
+  ];
+
+  meta = with lib; {
+    description = "Simple command line Bluetooth LE scanner";
+    homepage = src.meta.homepage;
```

Much easier for the reviews who are processing update and alike if it's possible to go to upstream's website which additional effort.

```suggestion
    homepage = "https://gitlab.com/jtaimisto/bluewalker";
```